.github/dependabot.yml

```diff
@@ -1,11 +1,18 @@
 version: 2
 updates:
-- package-ecosystem: cargo
-  directory: '/'
-  schedule:
-    interval: weekly
-    day: monday
-    time: '12:00'
-  open-pull-requests-limit: 20
-  labels:
-  - dependencies
+
```

Comment:
Is my reading of these changes correct in that other than removing the label the only other changes are whitespace differences?

Reply:
yeah I think its more usual and slightly clearer to indent yaml arrays, although

```
array:
  - one
  - two
  - three
```

and

```
array:
- one
- two
- three
```

are both valid.